src/tjoperators.jl

```diff
@@ -0,0 +1,640 @@
+#= Operators that act on t-J-type models
+i.e. the local hilbert space consists of 
+
+- usual basis states: 
+    |∅⟩, |↑⟩, |↓⟩
+- slave-fermion basis states (c_σ = h† b_σ; holon h is fermionic, spinon b_σ is bosonic): 
+    |h⟩ = h†|∅⟩, |↑'⟩ = (b↑)†|∅⟩, |↓'⟩ = (b↓)†|∅⟩
+=#
+module TJOperators
+
+using TensorKit
+
+export tj_space
+export c_num, u_num, d_num, c_num_hole
+export S_x, S_y, S_z, S_plus, S_min
+export u_plus_u_min, d_plus_d_min
+export u_min_u_plus, d_min_d_plus
+export u_min_d_min, d_min_u_min
+export c_plus_c_min, c_min_c_plus, c_singlet
+export S_plusmin, S_minplus, S_exchange
+
+export nꜛ, nꜜ, nʰ
```

Comment:
```suggestion
export nꜛ, nꜜ, nʰ, n
```
There is no namespace conflict here so I'd just export this.